### PR TITLE
Added missing SAML subsections to /concepts sidebar 

### PR DESCRIPTION
**Pull request by AndriiDovhopolov-okta**
## Description:
- Missing SAML subsections added
image
- Refactored /concepts sidebar config  - should be discussed. 
- **Is this PR related to a Monolith release?** No
### Resolves:

* [OKTA-320796](https://oktainc.atlassian.net/browse/OKTA-320796)

### CODE REVIEW

File: packages/@okta/vuepress-site/.vuepress/nav/concepts.js [hunk 1]
```diff
@@ -2,74 +2,88 @@
...
+        path: '/docs/concepts/policies/'
+    },
     { 
-        title: 'SCIM', 
-        link: '/docs/concepts/scim/'
+      title: 'Social Login Overview', 
+      path: '/docs/concepts/social-login/'
+    },  
+    { 
+        title: 'Understandign SAML',
```
Comment (denysoblohin-okta) [inline packages/@okta/vuepress-site/.vuepress/nav/concepts.js:73]:
```suggestion
        title: 'Understanding SAML',
```
typo

**Comment by albertchen-okta** [comment]
looks good, can we also bring the SCIM concept FAQ into the tree nav? https://developer.okta.com/docs/concepts/scim/faqs/

**Review by oleksandrpravosudko-okta — APPROVED** [review]
nit-pick: it looks worth aligning `Overview` case for SAML/SCIM/Social Login

**Comment by AndriiDovhopolov-okta** [comment]
image
Current state - fixed Overview first letter, added SCIM

**Pull request merged**